Code/Engine/Foundation/Profiling/Profiling.h

```diff
@@ -164,25 +164,25 @@ class EZ_FOUNDATION_DLL ezProfilingSystem
 };
 
 #if EZ_ENABLED(EZ_USE_PROFILING) || defined(EZ_DOCS)
-
+#  ifndef TRACY_ENABLE
 /// \brief Profiles the current scope using the given name.
 ///
 /// It is allowed to nest EZ_PROFILE_SCOPE, also with EZ_PROFILE_LIST_SCOPE. However EZ_PROFILE_SCOPE should start and end within the same list scope
 /// section.
 ///
 /// \note The name string must not be destroyed before the current scope ends.
 ///
-/// \sa ezProfilingScope
+/// \sa nsProfilingScope
```

Comment:
What is that ns prefix doing here?

Reply:
Hmm, left over from porting it over to the Ez clone. That's my bad. I'm not at my pc currently, but I can tackle these issues soon.

Reply:
Fixed